### PR TITLE
立法會投票光譜 - 開放數據分析

### PR DESCRIPTION
**Pull request by debuggingfuture**
https://code4hk.hackpad.com/VxmFIgX53qb

### CODE REVIEW

**Comment by gilbertwat** [comment]
Just a nitpick, should we add the image to our repo?

File: _posts/2014-04-26-legco-votes-analysis.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+而開放數據優點在於每人都可以參與分析，基於數據理性討論，互相進步。立法會今次起步是一好開始。促請政府今後開放更多適合電腦處理的數據。有興趣可到[課程網站](https://course.ie.cuhk.edu.hk/~engg4030/tutorial/)了解更多及嘗試參考例子分析；這次分析亦為開源項目設於[Github](http://nbviewer.ipython.org/urls/raw.githubusercontent.com/hupili/legcohk/master/LegCoHK.ipynb)。另外[Code for Hong Kong](https://www.facebook.com/groups/code4hk/) 和 [Open Data Hong Kong](http://odhk.github.io/)均會舉辦Hackathon等相關活動，使大家能實際應用立法會及其他開放數據，可密切留意！
+
+＊主成分分析，見[維基百料](http://zh.wikipedia.org/zh-hk/%E4%B8%BB%E6%88%90%E5%88%86%E5%88%86%E6%9E%90) 
+
+＊[立法會xml 投票紀綠](http://www.legco.gov.hk/general/english/counmtg/yr12-16/mtg_1213.htm#cm20121010) 
+
+＊課程分析結果詳請，以Creative Commons 4.0 發佈
+
+[LegCoHK](http://nbviewer.ipython.org/urls/raw.githubusercontent.com/hupili/legcohk/master/LegCoHK.ipynb)  [1 PCA](http://bit.ly/1riabfV) [2 Recommender Analysis](http://nbviewer.ipython.org/urls/course.ie.cuhk.edu.hk/~engg4030/tutorial/tutorial9/Recommender-System.ipynb) [3 Graph Analysis](http://nbviewer.ipython.org/urls/course.ie.cuhk.edu.hk/~engg4030/tutorial/tutorial10/Graph-Analysis.ipynb)
```
Comment (hupili) [inline _posts/2014-04-26-legco-votes-analysis.md]: The four links look mixed together. Suggest a bullet list:

```
   * [LegCoHK](http://nbviewer.ipython.org/urls/raw.githubusercontent.com/hupili/legcohk/master/LegCoHK.ipynb)  
   * [1 PCA](http://nbviewer.ipython.org/urls/course.ie.cuhk.edu.hk/~engg4030/tutorial/tutorial7/Dimensionality-Reduction.ipynb) 
   * [2 Recommender Analysis](http://nbviewer.ipython.org/urls/course.ie.cuhk.edu.hk/~engg4030/tutorial/tutorial9/Recommender-System.ipynb) 
   * [3 Graph Analysis](http://nbviewer.ipython.org/urls/course.ie.cuhk.edu.hk/~engg4030/tutorial/tutorial10/Graph-Analysis.ipynb)
```

**Comment by debuggingfuture** [comment]
Good idea to add image, but seems not that convenient to do so in github site.
trying to copy and paste in comment, not sure will it be housekeep

image

File: _posts/2014-04-26-legco-votes-analysis.md (hunk 1 above)
Comment (debuggingfuture) [inline _posts/2014-04-26-legco-votes-analysis.md]: :+1:

**Comment by hupili** [comment]
We can put images in the same repo, e.g. `images/xxxx` and refer using http://blog.code4.hk/images/xxxx

Then our Git repo is self contained. It's safe to be moved to other places.

Even better is to define in `_config.yaml` a variable, e.g. `SITE_PREFIX`. We can link to the image in our md source by:

```
{{SITE_PREFIX}}/images/xxxx
```

It is more portable in this way.

**Comment by debuggingfuture** [comment]
googled seems can't upload image to github repo without leaving the browser.
this might be one useful hack
http://solutionoptimist.com/2013/12/28/awesome-github-tricks/

**Pull request merged**